### PR TITLE
Make blood splatters fly for longer

### PR DESCRIPTION
**Pull request by karmaisblackandbluepilled**
[Guidelines]: # (Be sure that your PR follows our guidelines, such as modularization and comment standards. You can read more about the subject here: https://github.com/HippieStation/HippieStation/blob/master/hippiestation/README.md )

## Changelog
:cl:
tweak: Blood sprays from when you get shot stay up for longer now.
/:cl:

## About The Pull Request
I think it looks Cooler™

## Why It's Good For The Game
looks cool, sorta like shit from a comic book
inshallah I will burn Marvel to the ground

### CODE REVIEW

File: hippiestation/code/game/objects/effects/decals/cleanable/humans.dm [hunk 1]
```diff
@@ -21,7 +21,7 @@
 /obj/effect/decal/cleanable/blood/hitsplatter/proc/GoTo(turf/T, var/range)
 	for(var/i in 1 to range)
 		step_towards(src,T)
-		sleep(1)
+		sleep(8)
```
Comment (yoyobatty) [inline hippiestation/code/game/objects/effects/decals/cleanable/humans.dm]: wrong

**Comment by QuarianCommando** [comment]
So this isn't talking about the new blood splatter effects right? This is talking about the red puff effect from getting shot?  Might want to look into how the new effect works and see if you can make it spray out more blood too.

**Comment by yoyobatty** [comment]
We're both discussing it, and to clarify, this the blood that flies away from being shot or melee, not the puff.

**Comment by Zyzarda** [comment]
+1 for bullets spraying blood on exit wound

**Comment by QuarianCommando** [comment]
Alright, nice.

**Comment by yoyobatty** [comment]
> +1 for bullets spraying blood on exit wound

it already does lol

**Comment by yoyobatty** [comment]
image
 
literally 1 change, zyz

**Comment by AnthonyJNgo** [comment]
+3 needs a few more votes before merge.

**Comment by QuarianCommando** [comment]
+1

**Comment by MrEpicSax** [comment]
+1 yes please

**Pull request merged**